pom.xml

```diff
@@ -174,11 +174,11 @@
             <groupId>io.jenkins.configuration-as-code</groupId>
             <artifactId>test-harness</artifactId>
             <scope>test</scope>
-            <!-- TODO bump version in jcasc -->
+            <!-- TODO bump parent pom or cherry-pick https://github.com/jenkinsci/junit-plugin/pull/362 -->
```

Comment:
an alternative to ignoring jenkins-test-harness from jcasc test-harness: bump parent pom and cherry-pick https://github.com/jenkinsci/junit-plugin/pull/362
```suggestion
            <!-- TODO bump parent pom and cherry-pick https://github.com/jenkinsci/junit-plugin/pull/362 -->
```